tools/exporter/exporter.py

```diff
@@ -358,12 +358,15 @@ def parse_command_line_arguments(logger):
     :param logger:  the logger
     :return:        config_filename passed as argument if any, else DEFAULT_CONFIG_FILENAME
                     export_formats passed as argument if any, else 'pdf'
+                    list_export_profiles if passed as argument, else None
+                    do_loop False if passed as argument, else True
     """
     parser = argparse.ArgumentParser()
     parser.add_argument('--config', help='config file to use, defaults to ' + DEFAULT_CONFIG_FILENAME)
     parser.add_argument('--format', nargs='*', help='formats to download, valid options are pdf, json, docx')
     parser.add_argument('--list_export_profiles', nargs='*', help='display all export profiles, or restrict to specific'
                                                                   ' template_id if supplied as additional argument')
+    parser.add_argument('--no_loop', nargs='*', help='execute once and terminate')
```

Comment:
Yep, quite easily.  I assumed the more common use case would be loop forever, so I wrote the command line argument for the atypical use case.  Will reverse.